<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilePage</class>
 <widget class="QWidget" name="ProfilePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="profileLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadHistoryButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>109</y>
     <width>96</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Show History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>550</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>57</y>
     <width>401</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="historyTableView">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>158</y>
     <width>511</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Click on Replay to view the game moves</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>550</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
